README.md

```diff
@@ -85,5 +85,86 @@ iex> EchoClient.echo(pid, "Hi")
 *DBG* #PID<0.379.0> received frame: {:text, "Hi"}
 ```
 
+#### start_link/3 && start/3 
+
+You can start a supervisioned WebSockex connection by using the regular **start_link/3**:
+
+```elixir
+iex> {:ok, pid} = WebSockex.start_link(url, __MODULE__, state)
+```
+
+And an unsupervisioned one (meaning it won't try to re-establish the connection on an exceptional close) with **start/3**:
+
+```elixir
+iex> {:ok, pid} = WebSockex.start(url, __MODULE__, state)
+```
+
+This can be useful if you have don't want the socket to automatically try reconnection, you have no need for the built-in supervision or if you're implementing your own supervision strategy.
+
+
+#### Callbacks
+###### terminate/2
```

Comment:
A couple things here.

1. I'd like Debugging to be the last section in the README just because it's kind of like a troubleshooting thing.
2. Maybe instead of documenting the callback itself, name the section **Tips** and then _Terminating with `:normal` after an Exceptional Close or Error_.
3. Match the [Installation][1] header levels. You could probably go down one level with the nested header, but make the top level header match.

[1]: https://github.com/Azolo/websockex#installation

Reply:
I also agree, Tips does sound more appropriate to what is being said